density/density.py

```diff
@@ -356,5 +356,44 @@ def capacity():
     return render_template('capacity.html', locations=locations)
 
 
+@app.route('/map')
+def map():
+    """ Render and show maps page """
+
+    # This part behaves like the capacity function
+    # Read capacity of groups from json file
+    with open('data/capacity_group.json') as json_data:
```

Comment:
this should be moved outside the route so that it is only performed once when the entire flask app starts.